pubspec.yaml

```diff
@@ -9,11 +9,11 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  collection: ^1.15.0
-  dio: ^4.0.0
-  http_parser: ^4.0.0
+  collection: ^1.17.1
```

Comment:
This bump isn't required and it prevents running tests on older SDKs, please revert to `collection: ^1.15.0`.

Reply:
This effectively requires Dart 2.18.0+ at the moment.

Reply:
The change is reverted.